### PR TITLE
chore: bump `unstructured-inference` to `0.6.3`

### PR DESCRIPTION
**Pull request by badGarnet**
Bump `unstructured-inference`:
- now uses `yolox_quantized` as default element detection model for pdf/image partition in `hi_res`
- dedupe nested elements
- allow setting inference parameters via environment variables

### CODE REVIEW

File: test_unstructured_ingest/expected-structured-output/biomed-api/65/11/main.PMC6312790.pdf.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
   {
-    "type": "UncategorizedText",
-    "element_id": "0e58869830c7b4461a4d1879223e4139",
+    "type": "Header",
+    "element_id": "c1f4b5ba045830c1866db8f8aa0b54ac",
```
Comment (badGarnet) [inline test_unstructured_ingest/expected-structured-output/biomed-api/65/11/main.PMC6312790.pdf.json:4]: due to using yolox -> better element classification

File: test_unstructured_ingest/expected-structured-output/biomed-api/65/11/main.PMC6312790.pdf.json [hunk 2]
```diff
@@ -139,6 +159,16 @@
...
+  {
+    "type": "NarrativeText",
+    "element_id": "62e4907f12a32a7b9ccd57ed477eb54a",
+    "metadata": {
+      "data_source": {},
+      "filetype": "application/pdf",
+      "page_number": 1
+    },
+    "text": "reactions responsible for corrosion and surface deterioration. © 2018 Published by Elsevier Inc. This is an open access article under the CC BY-NC-ND license"
+  },
```
Comment (badGarnet) [inline test_unstructured_ingest/expected-structured-output/biomed-api/65/11/main.PMC6312790.pdf.json:171]: seems to be part duplicated text, cc @benjats07 for double check this behavior

File: test_unstructured_ingest/expected-structured-output/local-single-file-with-pdf-infer-table-structure/layout-parser-paper.pdf.json [hunk 3]
```diff
@@ -171,13 +171,13 @@
   },
   {
     "type": "NarrativeText",
-    "element_id": "1f0f5df7c23d4f8e8de4de3085abd7d8",
+    "element_id": "836e6ef5cecc9a73356c0d5bee181829",
```
Comment (badGarnet) [inline test_unstructured_ingest/expected-structured-output/local-single-file-with-pdf-infer-table-structure/layout-parser-paper.pdf.json:174]: this is just swapped in location with the element below

**Review by cragwolfe — CHANGES_REQUESTED** [review]
This bump is 3 minor versions behind the latest `unstructured-inference==0.6.6` (as of writing) so I think the priority is getting to parity. To do that, I'd propose two PR's:

* update this PR to bump to `unstructured-inference==0.6.6` changing the default model name to `detectron2_onnx`. this will isolate diff's in fixtures not related to updating the model
* once merged, create another PR to update the default model. then, carefully examine the ingest test diffs. IRS-form-1987.png is one I'd focus on first.

CC @christinestraub as https://github.com/Unstructured-IO/unstructured/issues/1233 may be necessary before the last bullet to keep the existing good natural-reading-order outputs that currently exist in fixture output.

**Comment by christinestraub** [comment]
@cragwolfe Created PR [#1560](https://github.com/Unstructured-IO/unstructured/pull/1560) for #1233.

**Comment by badGarnet** [comment]
breaking this task with other PRs.

**Pull request closed without merging**